### PR TITLE
fixed issue on checkbox_list event data changed and also in validity …

### PR DESCRIPTION
**Pull request by jvervier**
Hello @richturner 
Found one little issue from an undefined case and another one with CHECKBOX_LIST value change return for this issue.
Ready to be reviewed

### CODE REVIEW

File: ui/component/or-mwc-components/src/or-mwc-input.ts [hunk 1]
```diff
@@ -1639,6 +1642,10 @@ export class OrMwcInput extends LitElement {
                 (searchableElement as HTMLInputElement).value = "";
             }
         }
+
+        if (this.type === InputType.CHECKBOX_LIST) {
+            this.dispatchEvent(new OrInputChangedEvent(newValue.join(','), previousValue, enterPressed));
```
Comment (richturner) [inline ui/component/or-mwc-components/src/or-mwc-input.ts:1647]: What's the reason to flatten the array into a comma separated string? Where ever the value event is consumed should expect an array value if the input is multiselect.

**Review by richturner — COMMENTED** [review]
Please see my comments

File: ui/component/or-mwc-components/src/or-mwc-input.ts [hunk 2]
```diff
@@ -70,7 +70,7 @@ export class OrInputChangedEvent extends CustomEvent<OrInputChangedEventDetail>
 
     public static readonly NAME = "or-mwc-input-changed";
 
-    constructor(value?: any, previousValue?: any, enterPressed?: boolean) {
+    constructor(value?: any | any[], previousValue?: any | any[], enterPressed?: boolean) {
```
[the remark this answers is not in the document]
Comment (jvervier) [reply ui/component/or-mwc-components/src/or-mwc-input.ts]: Just checked, no need so I removed

File: ui/component/or-mwc-components/src/or-mwc-input.ts (hunk 1 above)
Comment (jvervier) [reply ui/component/or-mwc-components/src/or-mwc-input.ts]: Not necessary anymore. In one of my try I moved to that and forget to clean it. It's updated now

**Comment by jvervier** [comment]
Hi @richturner , I took your comments in hands and now pushed a new version :-)

**Comment by jvervier** [comment]
Hi @richturner, Cleaning done for added array

**Pull request merged**